prober/prober.dockerfile

```diff
@@ -4,7 +4,6 @@ FROM cgr.dev/chainguard/bash:latest
 COPY jvsctl /jvsctl
 
 COPY ./prober/prober.sh /prober.sh
-RUN chmod +x /prober.sh
```

Comment:
Do we no longer need to mark the prober script as executable?

Reply:
We will make the file executable locally, and when we copy it into container it stays executable.